ReactNativeClient/lib/models/Folder.js

```diff
@@ -19,12 +19,14 @@ class Folder extends BaseItem {
 		return {
 			id: null,
 			title: '',
+			icon: '',
```

Comment:
Not needed, please remove.

Reply:
Taken directly from the spec:

> * Add a new icon property to the Folder object - it will be a string that will take any of the Fork Awesome icon names.

I'm sorry that I misread object as class and not object in memory. I don't understand why this needs to be removed, so pardon my delay while I work on learning the basics. I'm definitely not from the same school of thought as you are and didn't know I didn't know the right term here.

Reply:
> Add a new icon property to the Folder object

I should have made this clearer - it means adding it to the database. Here you don't need to add anything (in fact I think this part of the code is a bit obsolete).